pants.toml

```diff
@@ -144,5 +144,27 @@ extra_requirements = [
 ]
 config = "lint-configs/python/.flake8"
 
+[pytest]
+lockfile = "lockfiles/pytest.lock"
+version = "pytest==7.0.1"  # copied from https://www.pantsbuild.org/v2.14/docs/reference-pytest#version
+extra_requirements.add = [
+  "pytest-benchmark[histogram]==3.4.1",  # used for st2common/benchmarks
+  #"pytest-timer[colorama]",  # report test timing (--with-timer ala nose-timer)
+  "pytest-icdiff",  # make diff output easier to read
+  "pygments",  # highlight code in tracebacks
+
+  # other possible plugins
+  #"pytest-timeout",  # time limit on tests
+  #"pytest-mock",  # more convenient mocking
+
+  # included by default with pants
+  #"pytest-cov",  # coverage
+  #"pytest-xdist",  # parallel test runs (pants uses this if [pytest].xdist_enabled)
```

Comment:
You won't be able to use some Pants features with these commented out

Reply:
This serves as documentation because I'm using `extra_requirements.add`